x-pack/plugins/security_solution/public/resolver/models/process_event.ts

```diff
@@ -88,3 +99,62 @@ export function uniqueParentPidForProcess(passedEvent: ResolverEvent): string |
     return passedEvent.process.parent?.entity_id;
   }
 }
+
+/**
+ * Returns the process event's parent pid
+ */
+export function processParentPid(passedEvent: ResolverEvent): number | undefined {
+  if (event.isLegacyEvent(passedEvent)) {
+    return passedEvent.endgame.ppid;
+  } else {
+    const ppid = passedEvent.process.parent?.pid;
+    return ppid ? ppid : undefined;
```

Comment:
Why not just `return passedEvent.process.parent?.pid`?
By doing `ppid ? ppid : undefined`, won't you end up showing `undefined` instead of `0` when the `ppid` is `0`?

Reply:
https://github.com/elastic/kibana/pull/69098/commits/44a4d5680c6f0e2f8acca7bab500048b310c6f9b